ui/jest.config.js

```diff
@@ -23,7 +23,7 @@ module.exports = {
     'ts-jest': {
       tsConfig: 'tsconfig.test.json',
       diagnostics: {
-        ignoreCodes: [6133] // ignore `'foo' is declared but its value is never read.`
+        ignoreCodes: [6133, 6192] // ignore unused variable errors
```

Comment:
what triggered the addition?

Reply:
```ts
import {necessaryButUnusedFunctionOne, necessaryButUnusedFunctionTwo} from 'foo'
```

triggers a `6192` which is "all imports must be used"